### ISSUE TITLE
Fit everything into a common style

### ISSUE THREAD

**Issue by dotSp0T** [dotSp0T/Anno_1800_Chains_Consumption#14]
Rework styling and design decisions

- make small icons larger or replace with symbols / clearer icons
- create section sections and dividers by DLC so ppl can leave out things they don't own when printing for example (or generating maybe?)
- could introduce some form of color-coding per DLC and/or pop-level

**Comment by kruzuahz** [comment]
Good ideas, you may want to consider [latex modes](https://wiki.contextgarden.net/Modes), These seem to work like c-style defines, I'm not sure if they work with pdflatex though.

We could also add DLC icons to the relevant sections

**Comment by dotSp0T** [comment]
DLC Icons seconded!

Also holy shit the whole document has become barely legible. There's too much going on on the prodchains.

Keeping this issue as a place to discuss and find stuff to do. Creating appropriate sub-issues with single todo tasks

**Comment by dotSp0T** [comment]
We could do sth similar to the ingame screen, where we got one block per pop-level, icons for all their needs, and 2 numbers next to each icon (how much is consumed per household, and how many more inhabitants will live in the house if fulfilled)

Obviously this will be a rough abbreviation of the whole mechanic. But the whole point of the prodchain document is to give overviews and lookup tables, not exact graphs, etc.

**Comment by kruzuahz** [comment]
I Agree, we could do this for the lifestyle needs only for now. thats already 13(tiers)x6(needs) items removed from the main chains, and reduces the amount of work.

**Comment by kruzuahz** [comment]
On a side note would you be OK with creating a release for the document as is? We might be able to get more collaborators if people could use an updated version.

**Comment by dotSp0T** [comment]
Oh yeah, let's do that. I sorta wanted to get a pipeline going to automatically create releases. But we can do a manual one (though at least have to fix #19 )

**Comment by kruzuahz** [comment]
great, I will try to look into that this week
